.env.example

```diff
@@ -13,4 +13,9 @@ SAILTHRU_SECRET=
 METAPHYSICS_URL=
 
 # Database auth
-MONGOHQ_URL=mongodb://localhost:27017/kaws
\ No newline at end of file
+MONGOHQ_URL=mongodb://localhost:27017/kaws
+
+# Google Sheets credentials
+GOOGLE_CLIENT_ID=REPLACE
+GOOGLE_CLIENT_SECRET=REPLACE
```

Comment:
The app will no longer boot unless these are present.

Reply:
where can these values be found?

Reply:
`hokusai staging env get` would get you the current staging config, including these env vars.